### ISSUE TITLE
Not a good way to pull out a package.

### ISSUE THREAD

**Issue by mshumakov** [vseinstrumentiru/dto-bundle#1]
`composer require vi-tech/dto-bundle` => `composer require vseinstrumentiru/dto-bundle`

**Comment by mshumakov** [comment]
@TemirkhanN fyi ^

**Comment by TemirkhanN** [comment]
@mshumakov do you mean namespace should be one instead of another because of the organization name?

**Comment by mshumakov** [comment]
@TemirkhanN yes! Since this package is positioned as an open package, we cannot pull it along the current path.

**Comment by max107** [comment]
ping

**Comment by TemirkhanN** [comment]
Will be fixed in next major release.
commit https://github.com/vseinstrumentiru/dto-bundle/commit/87cd2043b17dcea567523d99eb1412efd1c69047

**Comment by TemirkhanN** [comment]
This is it, boys. They finally introduced very similar feature in symfony 6.3. 
https://symfony.com/blog/new-in-symfony-6-3-mapping-request-data-to-typed-objects
